python/neuroglancer/pipeline/storage.py

```diff
@@ -107,6 +117,50 @@ def get_file(self, file_path):
             content = self._maybe_uncompress(content)
         return content
 
+    def get_files(self, file_paths):
+        """
+        returns a list of files faster by using threads
+        """
+
+        file_paths = deque(file_paths)
+
+        results = []
```

Comment:
Hmm, this is an interesting idea. I didn't know you could do a generate over python threads. However, wouldn't a `yield` statement pause the download of all the items on a given thread until the result is consumed? The upside is that you limit the amount of memory used to a factor proportional to n_threads, but I don't think that's a concern in this case.